lib/imapSimple.js

```diff
@@ -115,7 +115,7 @@ ImapSimple.prototype.openBox = function (boxName, callback) {
 ImapSimple.prototype.search = function (searchCriteria, fetchOptions, callback) {
     var self = this;
 
-    if (!callback && typeof(fetchOptions) === 'function') {
+    if (!callback && typeof (fetchOptions) === 'function') {
```

Comment:
i agree with this change, but the parens should be removed because they're redundant